### ISSUE TITLE
[Accessibility] In high contrast mode, boundaries and icons of the controls are not visible.

### ISSUE THREAD

**Issue by xtinae** [swagger-api/swagger-editor#2411]
### Q&A 
 - OS: Windows
 - Browser: Edge
 - Version:  88.0.692.0 (Official build) dev (64-bit)
 - Method of installation: npm
 - Swagger-Editor version: 3.15.2
 - Swagger/OpenAPI version: N/A

### Describe the bug you're encountering
Boundary of editor and other controls are not visible in high contrast black/white mode.
Dropdown icon of "Schemes" control is not visible in high contrast black mode.

If boundaries and icons of controls are not visible in high contrast mode, it becomes an issue for the visually impaired user to identify controls and navigate the page in high contrast mode.

### To reproduce...
 System-Settings>Ease of access>High contrast>White/Black
Go to https://editor.swagger.io/ 

### Expected behavior
Boundary of editor and other controls should be visible in high contrast black/white mode.
Dropdown icon of "Schemes" control should be visible in high contrast black mode.

### Screenshots
Normal mode
![image](https://user-images.githubusercontent.com/8244914/107102133-57dda700-67ce-11eb-80fb-caf3a22c0625.png)

High Contrast - Black mode
![image](https://user-images.githubusercontent.com/8244914/107102097-40062300-67ce-11eb-9b9e-387342d454a2.png)

High Contrast - White mode
![image](https://user-images.githubusercontent.com/8244914/107102382-37fab300-67cf-11eb-9787-8b1b1c401145.png)

### Additional context or thoughts
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/02.01.02_60/en_301549v020102p.pdf

**Comment by sukoorel** [comment]
Hello, 

Just checking if there is any update on this issue?

Thank you.

**Comment by MichakrawSB** [comment]
We’ve released Swagger Editor v5! 🎉 Check out the details here: [Inside the New Swagger Editor](https://swagger.io/blog/inside-the-new-swagger-editor/).
We’re closing old issues related to previous versions. If you think any of them are still relevant, please open a new issue – this helps us prioritize what matters most. Thanks for your activity! 🚀
